Changelog.md

```diff
@@ -17,6 +17,7 @@ Bugfixes:
  * Control Flow Graph: Perform proper virtual lookup for modifiers for uninitialized variable and unreachable code analysis.
  * Immutables: Fix wrong error when the constructor of a base contract uses ``return`` and the parent contract contains immutable variables.
  * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
+ * IR Generator: Fix ICE when copying reference types in calldata and storage to struct or array members in memory.
```

Comment:
@ekpyron Changed to ICE.

Reply:
Actually, now it isn't sorted alphabetically anymore ;-D.

Reply:
Fixed!